### ISSUE TITLE
Smarther Thermostat mismatch when i have more than one in HA

### ISSUE THREAD

**Issue by exico91** [sdomotica/hassio-addons#7]
I have 3 Smarther Thermostat in this establishment.

So, when I installed the plugin i noticed that the thermostat interface was fine as you can see in the first image but the 3 status windows all report the status of just the first thermostat as you can see in the second image.

It's possibile to fix this?

![First - thermostat modules](https://user-images.githubusercontent.com/97479177/188265932-66b0a3b1-1171-4f31-bd73-96abed7e578a.png)

![Second - Status modules](https://user-images.githubusercontent.com/97479177/188265935-89c01096-c21f-42dc-916f-f1c6db2d19c2.png)

**Comment by sdomotica** [comment]
Did you are using the last Addon version?
If not, please use it and rebuild all
![immagine](https://user-images.githubusercontent.com/781294/188266251-03f69b1d-6866-46b0-a254-0f6a88f6cbce.png)
These are two Smarther2 and works fine
![immagine](https://user-images.githubusercontent.com/781294/188266291-d06dc11a-3afb-41f2-b1a0-05f4b4af2dd1.png)

Or you are using Smarther1?

**Comment by exico91** [comment]
I believe it's a version 2, not sure. I just did a Rebuild / Ricostruisci in the addon but nothing changed and yes, I'm using the latest version 2022.08.1 of Sdomotica Home + Control 2022.
Also HA is on the latest version: Home Assistant 2022.8.7 Supervisor 2022.08.5 Operating System 8.5 Frontend 20220802.0 - latest

**Comment by sdomotica** [comment]
You have Smarther2. Did you have different rooms for each smarther into Home Control app?

**Comment by exico91** [comment]
Yes, different rooms

**Comment by sdomotica** [comment]
Delete the package, unistall addon and reinstall it.

**Comment by exico91** [comment]
Sorry, i had time just now to try and fix this.
I removed the addon, removed the package and the extra code in configuration.yaml.

I had to log via ssh inot HA and delete a dangling docker volume because the addon didnt want to start after the reinstall and the logs said that there was a error in loading a config file in /data

After a login via ssh i resoled with "docker volume rm $(docker volume ls -qf dangling=true)"

Anyway now is worse than before. Now i have 6 thermostat interfaces and still the status windows are as before
![smarther](https://user-images.githubusercontent.com/97479177/189477012-14370e9e-d897-4d47-9a4d-fb422e080fdd.jpg)

But if i check in entities i can see that there are 3 different entities for the 3 thermostats

I dont know why but checking the package i can see that the groups get data from the same thermostat:

![thermo2](https://user-images.githubusercontent.com/97479177/189477149-549d6703-feff-4bfd-8333-a6b0ad8a42b8.png)

Oh, I noticed that now i do not have any control. If i switch a light it doesnt turn off or on that light, also the statuses are stuck.

I guess that its better do a complete reinstall of HA at this point
